app/assets/javascripts/discourse/tests/acceptance/do-not-disturb-test.js

```diff
@@ -112,8 +112,7 @@ acceptance("Do not disturb", function (needs) {
   });
 });
 
-acceptance("Do not disturb - new user menu", function (needs) {
-  needs.user({ redesigned_user_menu_enabled: true });
+acceptance("Do not disturb - user menu", function (needs) {
```

Comment:
I suspect we'll need to update some of the other tests in this file so that they work with the new user menu. (or, if they're now duplicated, we can delete them)